pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1001,9 +1008,9 @@ public static String writeRequest(@NonNull String instanceId, @NonNull String dd
                 } catch (Exception e) {
                     //This error will trigger on studies with no participants, this skips
                     //the error log if that is the reason for the upsert failure.
-                        logger.error(String.format("Error inserting newly created kit request shipping with dsm kit request id: %s in "
+                    logger.error(String.format("Error inserting newly created kit request shipping with dsm kit request id: %s in "
```

Comment:
Sorry, my comment was not clear.
I'm concerned about converting the exception to a log statement. Generally we would only do that at the interface point -- for example, http request handling, or pusbsub message handling, etc. This method appears to be called from a handful of different places, including a route. In that case the right thing to is let the callers handle the exception as they see fit, rather than swallowing it here.
I realize the form of this code already existed, and this is an opportunity for improvement. But I appreciate that is not technically in scope for this PR.